packages/element/tests/cropElement.test.tsx

```diff
@@ -3,21 +3,30 @@ import { vi } from "vitest";
 
 import { KEYS, cloneJSON } from "@excalidraw/common";
 
-import { duplicateElement } from "@excalidraw/element/duplicate";
-
-import type {
-  ExcalidrawImageElement,
-  ImageCrop,
-} from "@excalidraw/element/types";
-
-import { Excalidraw, exportToCanvas, exportToSvg } from "..";
-import { actionFlipHorizontal, actionFlipVertical } from "../actions";
-
-import { API } from "./helpers/api";
-import { Keyboard, Pointer, UI } from "./helpers/ui";
-import { act, GlobalTestState, render, unmountComponent } from "./test-utils";
-
-import type { NormalizedZoomValue } from "../types";
+import {
+  Excalidraw,
+  exportToCanvas,
+  exportToSvg,
+} from "@excalidraw/excalidraw";
+import {
+  actionFlipHorizontal,
+  actionFlipVertical,
+} from "@excalidraw/excalidraw/actions";
+
+import { API } from "@excalidraw/excalidraw/tests/helpers/api";
+import { Keyboard, Pointer, UI } from "@excalidraw/excalidraw/tests/helpers/ui";
```

Comment:
why do we import from `@excalidraw/excalidraw` here? And more importantly, we shouldn't since this isn't public API?